resources/scripts/quick_start.sh

```diff
@@ -52,40 +53,53 @@ print_message "" "" ""
 print_message "" "    Let's find out if your system has what it takes to run Warnet..." ""
 print_message "" "" ""
 
-minikube_path=$(command -v minikube || true)
-if [ -n "$minikube_path" ]; then
-    print_partial_message " ⭐️ Found " "minikube" ": $minikube_path " "$BOLD"
-else
-    print_partial_message " 💥 Could not find " "minikube" ". Please follow this link to install it..." "$BOLD"
-    print_message "" "   https://minikube.sigs.k8s.io/docs/start/" "$BOLD"
-    exit 127
-fi
-
-kubectl_path=$(command -v kubectl || true)
-if [ -n "$kubectl_path" ]; then
-    print_partial_message " ⭐️ Found " "kubectl" ": $kubectl_path " "$BOLD"
-else
-    print_partial_message " 💥 Could not find " "kubectl" ". Please follow this link to install it..." "$BOLD"
-    print_message "" "   https://kubernetes.io/docs/tasks/tools/" "$BOLD"
-    exit 127
-fi
-
 docker_path=$(command -v docker || true)
 if [ -n "$docker_path" ]; then
     print_partial_message " ⭐️ Found " "docker" ": $docker_path" "$BOLD"
 else
     print_partial_message " 💥 Could not find " "docker" ". Please follow this link to install Docker Engine..." "$BOLD"
     print_message "" "   https://docs.docker.com/engine/install/" "$BOLD"
-    exit 127
+    ERROR_CODE=127
 fi
 
 current_user=$(whoami)
+current_context=$(docker context show)
 if id -nG "$current_user" | grep -qw "docker"; then
     print_partial_message " ⭐️ Found " "$current_user" " in the docker group" "$BOLD"
+elif [ "$current_context" == "rootless" ]; then
+    print_message " " "⭐️ Running Docker as rootless" "$BOLD"
+elif [[ "$(uname)" == "Darwin" ]]; then
+    print_message " " "⭐️ Running Docker on Darwin" "$BOLD"
 else
     print_partial_message " 💥 Could not find " "$current_user" " in the docker group. Please add it like this..." "$BOLD"
     print_message "" "   sudo usermod -aG docker $current_user && newgrp docker" "$BOLD"
-    exit 1
+    ERROR_CODE=1
+fi
+
+minikube_path=$(command -v minikube || true)
+if [[ "$(uname)" == "Darwin" ]] && command -v minikube &> /dev/null && [[ "$(minikube version --short)" == "v1.33.1" ]]; then
+    if docker context ls | grep -q "docker-desktop"; then
+        print_message " ⭐️ Found " "Docker Desktop" "$BOLD"
+    else
+        print_partial_message " 💥 Could not find " "minikube version > 1.33.1" ". Please upgrade..." "$BOLD"
```

Comment:
I saw this error too, I thought it was going to be OK to use docker desktop instead?

Reply:
What does `docker context ls` return? 

Also, I have now added a check for Docker Desktop.